### PR TITLE
Ported Android SDK intro and re-org to all SDKs

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
Staging:

iOS intro: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/sectionOrgIntros/ios.html

Node intro: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/sectionOrgIntros/node.html

.NET intro: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/sectionOrgIntros/dotnet.html

React Native intro: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/sectionOrgIntros/react-native.html

Browser intro: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/sectionOrgIntros/browser.html (left this one alone mostly because we need to wrangle this section some more first)

### CODE REVIEW

File: source/graphql.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .. _graphql-api:
+.. _browser-intro:
```
Comment (cbush) [inline source/graphql.txt:2]: Why this change? Won't this conflict with the other browser-intro ref?

File: source/browser.txt [hunk 2]
```diff
@@ -1,10 +1,24 @@
-===
-Web
-===
+.. _browser-intro:
+
+=========================
+MongoDB Realm Browser SDK
+=========================
+
+To learn how to connect to the GraphQL API from a client application,
```
Comment (cbush) [inline source/browser.txt:7]: It would be nice if you could add a lead-in paragraph about what the Browser SDK _is_ in general, as you have on the other pages.

File: source/graphql.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/graphql.txt:2]: this was a mistake, was sorting out the graphql section and the web/browser one

File: source/browser.txt (hunk 2 above)
Comment (nathan-contino-mongo) [reply source/browser.txt:7]: Will do!
